### ISSUE TITLE
Training loss becomes NaN around epoch 3 when using VoVNet backbone

### ISSUE THREAD

**Issue by kimdonggue** [William-Yao-2000/DriveSuprim#4]
Hello, thank you for sharing your great work!

I am currently training the model using the VoVNet backbone with below command. I confirmed that loading the pretrained backbone was successful.

bash scripts/drivesuprim/training/rot_30-p_0.5/train.sh \
  drivesuprim_agent_vov 1 3 256

However, I am encountering an issue where all loss values suddenly become NaN around the 3rd epoch.

I haven't made any modifications to the training code, so I am not sure why this instability is occurring. Could you please check if there are any known issues kind of this , or provide some advice on how to resolve this?

thank you.
